dla_cnn/desi/train_v2.py

```diff
@@ -0,0 +1,216 @@
+""" Methods for training the Model"""
```

Comment:
This code looks good, but save it for a new PR.

Let's finish the current PR first.  Thanks